Source/JavaScriptCore/jit/AssemblyHelpers.h

```diff
@@ -1701,9 +1701,9 @@ class AssemblyHelpers : public MacroAssembler {
         ok.link(this);
     }
 
-    JS_EXPORT_PRIVATE void cageWithoutUntagging(Gigacage::Kind, GPRReg storage);
+    JS_EXPORT_PRIVATE void cageWithoutUntagging(Gigacage::Kind, GPRReg storage, bool nonNull = false);
```

Comment:
Instead of `nonNull`, can you use `mayBeNull` instead and default it to true?  This way, we don't have to reason about a lot of double negatives.

Reply:
Changed.